src/webgl/p5.Camera.js

```diff
@@ -193,23 +193,81 @@ p5.prototype.perspective = function (...args) {
   return this;
 };
 
+
 /**
  *
  * Enable or disable perspective for lines in the WebGL renderer.
- * When linePerspective is enabled, lines will be affected by the current camera's perspective.
- * When linePerspective is disabled, lines will have a uniform scale regardless of the camera's perspective.
+ *
+ * The behavior of `linePerspective()`  is associated with the type of camera projection being used.
+ *
+ *  When using `perspective()`, which simulates realistic perspective, linePerspective
+ *  is set to `true` by default. This means that lines will be affected by the current
+ *  camera's perspective, resulting in a more natural appearance.
+ *
+ *  When using `ortho()` or `frustum()`, which do not simulate realistic perspective,
+ *  linePerspective is set to `false` by default. In this case, lines will have a uniform
+ *  scale regardless of the camera's perspective, providing a more predictable and
+ *  consistent appearance.
+ *
+ *  You can override the default behavior by explicitly calling `linePerspective()` after
+ *  using perspective(), ortho(), or frustum(). This allows you to customize the line
+ *  perspective based on your specific requirements.
  *
  * @method linePerspective
  * @memberof p5.prototype
  * @param {boolean} enable - Set to `true` to enable line perspective, `false` to disable.
+ * @return {boolean} The boolean value representing the current state of linePerspective().
```

Comment:
For other p5 methods that could either be a setter or a getter, we just do one of them in the doc comment (e.g. here, having a `@param` and no `@return`) and then adding a second doc comment right after the first with the alternate overload (e.g. a `@return` with no `@param`.) An example of that is here: https://github.com/processing/p5.js/blob/37d3324b457b177319ed65468201f2806a66eff5/src/core/environment.js#L368-L371

Can we do the same thing here too?